packages/desktop-client/src/components/reports/DateRange.tsx

```diff
@@ -1,16 +1,21 @@
-import React from 'react';
+import React, { type ReactElement } from 'react';
 
 import * as d from 'date-fns';
 
 import { theme } from '../../style';
 import Block from '../common/Block';
 
-function DateRange({ start, end }) {
-  start = d.parseISO(start);
-  end = d.parseISO(end);
+type DateRangeProps = {
+  start: string | Date;
+  end: string | Date;
+};
 
-  let content;
-  if (start.getYear() !== end.getYear()) {
+function DateRange({ start, end }: DateRangeProps): ReactElement {
+  start = d.parseISO(start as string);
+  end = d.parseISO(end as string);
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏here's how we can solve it

```suggestion
function DateRange({ start: startProp, end: endProp }: DateRangeProps): ReactElement {
  const start = d.parseISO(startProp);
  const end = d.parseISO(endProp);
```

Reply:
Ahhh, that's a good call! Thank you so much!! 🙏
Pushed a fix to the PR!